### DISCUSSION TITLE
option to add ingredients to shopping list

### DISCUSSION THREAD

**Discussion by Narflar** [mealie-recipes/mealie#894] [Feature Request]
**Before submitting this feature request I have**
- [X] Reviewed the documentation and verified that my feature request isn't already a feature.
- [X] Reviewed the existing feature requests and my feature has not been requested.
- [X] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
when inside a recipe, it would be nice to be able to add the ingredients to a shopping list.  I don't have much need for the meal plan but I would like to make a shopping list out of a number of recipes.  

**Additional Information**
_FILL THESE OUT_
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature
           I don't even know where to begin with this...

**Comment by hay-kot** [comment]
Please use the feature request template. See below.

- https://github.com/hay-kot/mealie/issues/317

**Reply by Narflar** [reply]
updated, sorry about that.

**Comment by hay-kot** [comment]
You can create a shopping list and add recipes from the shopping list. It's not from the recipe, but it does fill the same use-case

**Reply by Narflar** [reply]
agreed, this does work but it's a lot of steps add to a list when you are browsing recipies.

**Reply by hay-kot** [reply]
> agreed, this does work but it's a lot of steps add to a list when you are browsing recipies.

Fair enough. The current implementation doesn't allow for easy extensibility. In the rewrite for v1 I've added this. Should see it in the next PR. 

![Screen Shot 2022-01-02 at 2 28 54 PM](https://user-images.githubusercontent.com/64056131/147892210-f8493c8b-e7bc-41db-b06f-e5409e8c917c.png)


![Screen Shot 2022-01-02 at 2 29 18 PM](https://user-images.githubusercontent.com/64056131/147892218-c6f7f752-6b82-41e8-adec-be24a2d0ce2a.png)
